mapr/tests/conftest.py

```diff
@@ -46,7 +46,7 @@ def mock_getconnection():
     def messages_iter():
         with open(os.path.join(common.HERE, 'fixtures', 'metrics.txt'), 'r') as f:
             for line in f:
-                msg = mock.MagicMock(error=lambda: None, value=lambda: line)
+                msg = mock.MagicMock(error=lambda: None, value=lambda line=line: line)
```

Comment:
I'm confused by this change, I'm not familiar with this syntax. Is it giving the lambda a name?

Reply:
It is solving this error: `B023 Function definition does not bind loop variable`